process/due-diligence-guidelines.md

```diff
@@ -82,7 +82,7 @@ the detail where necessary.
 * What are the most important holes? No High-Availability? No flow control? Inadequate integration points?
 * Code quality.  Does it look good, bad or mediocre to you (based on a spot review).  How thorough are the code reviews? Substance over form.
   Are there explicit coding guidelines for the project?
-* Dependencies.  What external dependencies exist, do they seem justified?
+* Dependencies.  What external dependencies exist, do they seem justified?  Note: all core dependencies should be listed in the document along with the details of relevant repos
```

Comment:
"core" implies "non-optional"?

Reply:
yes - that was the intent - happy to change it if there is better wording.

Reply:
"non-optional" seems better otherwise folks will debate what "core" is/means :)

Reply:
I don't know how to get the terminology right here but I think we need to distinguish between significant functional dependences, and programming language / package dependencies. We don't want to be reviewing giant lists of, say, Go packages or Node dependencies, but we do want to know when a project assumes the existence of a significant external component.